docs/user_docs/overview/supported-addons.md

```diff
@@ -8,107 +8,217 @@ sidebar_label: Supported addons
-| victoria-metrics  | VictoriaMetrics is a fast, cost-effective and scalable monitoring solution and time series database. |
-| weaviate        | Weaviate is an open-source vector database.                                                                                                                                                                       |
-| xinference      | Xorbits Inference(Xinference) is a powerful and versatile library designed to serve language, speech recognition, and multimodal models. |
-| yanshan         | YashanDB is a database management system developed by Shenzhen Institute of Computing Science. |
-| zookeeper       | Apache ZooKeeper is a centralized service for maintaining configuration information, naming, providing distributed synchronization, and providing group services. |
-
-## Supported functions of addons
+## Relational Databases
+
+MySQL and PostgreSQL are the two most popular open-source relational databases in the world, and they have branches/variants.
+
+### MySQL and its variants
+
+**Addon List**
+
+| Addons          | Description                                                                                                                                                                                                                                                                                                                                                             |
+|:----------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| mysql           | This addon uses the community edition MySQL image officially released by Oracle.                                                                                                                                                                                                                                                                                        |
+| apecloud-mysql  | ApeCloud MySQL is a free, fully compatible drop-in replacement for MySQL Community Edition, offering enhanced high availability through a RAFT protocol replication plugin. The image is provided by ApeCloud. Additionally, ApeCloud MySQL includes an open-source proxy called WeScale, which provides features such as read-write splitting and connection pooling.  |
+| mariadb         | MariaDB is a high performance open source relational database management system that is widely used for web and application servers.                                                                                                                                                                                                                                    |
+**Supported Features**
```

Comment:
Add a blank line before this line. It doesn't render well on GitHub page.

Reply:
great suggestion